### ISSUE TITLE
Add Ability to Capture Socket Service Banner

### ISSUE THREAD

**Issue by ikiril01** [CybOXProject/schemas#387]
We should consider adding the ability to the Network Socket Object to capture the banner message of the service that may be running on the socket. E.g., "SSH-2.0-OpenSSH_5.9p1"

**Comment by gtback** [comment]
The banner doesn't run on the socket. If anything, it is a property of a process listening on a socket, or on a connection to a socket.

**Comment by ikiril01** [comment]
@gtback good point! I think in most cases it's likely to be associated with the process listening on a socket. I'm wondering if this should instead be a field on the Process Object (if it makes sense at all).
